### PR TITLE
[2279] Remove unneeded sleep calls, leave imports for other functions (4-3-stable)

### PR DESCRIPTION
**Pull request by MartinFlores751**
Remove unneeded sleep calls. Ran core tests and all passed.

### CODE REVIEW

**Review by korydraughn — DISMISSED** [review]

**Comment by korydraughn** [comment]
Let's wait for one more approval before adding the pound.

**Comment by trel** [comment]
this is just the cherry-pick, yes?   seems okay upon inspection.

**Review by alanking — DISMISSED** [review]

**Review by korydraughn — APPROVED** [review]
Looks good to me.

**Review by alanking — APPROVED** [review]

**Pull request merged**
